frontend/components/product-list/MetaTags.tsx

```diff
@@ -13,7 +13,7 @@ export function MetaTags({ productList }: MetaTagsProps) {
    const searchParams = useSearchParams();
    const isFiltered =
       searchParams.query.length > 0 || searchParams.filters.allIds.length > 0;
-   let title = `iFixit | ${productList.title}`;
+   let title = `${productList.title} | iFixit`;
    if (!isFiltered && searchParams.page > 1) {
       title += ` - Page ${searchParams.page}`;
    }
```

Comment:
Taking into account pagination, I'd suggest having `All Parts - Page 2 | iFixit` instead of `All Parts | iFixit - Page 2`.
What do you think?

```suggestion
   let title = productList.title;
   if (!isFiltered && searchParams.page > 1) {
      title += ` - Page ${searchParams.page}`;
   }
   title += ' | iFixit';
```

Reply:
Your suggestion looks much better, actually! It didn't occur to me how weird `All Parts | iFixit - Page 2` looks until seeing those two arrangements side by side. Let's go with yours.